initiator.go

```diff
@@ -437,51 +447,60 @@ func (s *IscsiTargetSession) ReScan() error {
 		}
 	}
 
-	contents, err := ioutil.ReadFile(matches[0])
-	if err != nil {
-		return err
-	}
+	found := false
```

Comment:
and all the matches will show up at the same time, so we won't miss any?

Reply:
so i gave a for loop with 10 iterations, each with a 30msec sleep. within this time period . the first instance when filepath.Glob succeeds, we catch that. Since the glob pattern has multiple wild cards, it seemed to work when i test ran it . It caught around 4 matches...I hope thats how this works..

Reply:
This loop is what's giving me a little bit of pause... but I'll merge it for now and perhaps work on it more later when I go to test it internally.